plasmapy/particles/particle_class.py

```diff
@@ -429,6 +464,8 @@ def __init__(
 
         self.__name__ = self.__repr__()
 
+##########
+
```

Comment:
Delete lines...[PEP8: there should be a single blank line between methods](https://www.python.org/dev/peps/pep-0008/#blank-lines)